interface/ceed-python/ceed.py

```diff
@@ -26,26 +26,26 @@
 # Ceed Enums
 # ------------------------------------------------------------------------------
 # CeedMemType
-MEM_HOST       = lib.CEED_MEM_HOST
-MEM_DEVICE     = lib.CEED_MEM_DEVICE
+CEED_MEM_HOST       = lib.CEED_MEM_HOST
+CEED_MEM_DEVICE     = lib.CEED_MEM_DEVICE
```

Comment:
It seems to me that this might as well match the globals in the other interfaces. Not sure why I used the shorter name at first.

Reply:
Writing `ceed.CEED_MEM_HOST` feels clunky.

Reply:
Also, does this parameter need to be explicit? Seems like it'd be a property of the array (NumPy or whatever). When getting an array, perhaps it should be specified (if/when we support an array container that can manage GPU arrays).

Reply:
We can set MEM_HOST as the default and then only call with MEM_DEVICE when we have an option to do so

Reply:
Yeah, keyword arguments are the way to go, and will make the Python syntax a lot cleaner than we can do in C.